manifests/client.pp

```diff
@@ -264,9 +302,10 @@
     notify      => Exec["generate ${name}.ovpn in ${server}"],
   }
 
+  $name_escaped = regsubst(regsubst($name, '\.', '\\.', 'G'), '@', '\\@', 'G')
```

Comment:
This is true.
It was required for us as we use full e-mail addresses in resource title. The former version works fine as long as simple user names are used (plain alphanumeric). In our case, Perl's regular expression handling would treat those characters special and in turns wont match properly.